.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,130 @@
+### Description
+
+<!-- Describe your changes in detail -->
+
+### Motivation & Context
```

Comment:
I'd regroup this part with the `Related Issues`

Reply:
Or move `Related Issues` up here or just before this paragraph ?